### PR TITLE
Fix lint issues in scripts

### PR DESCRIPTION
**Pull request by pixeldesu**
# What

This PR fixes lint issues in the scripts directory (as of opening this PR from 770 -> 696)

# Why

There's over 900 lint errors in client and they should be cleared off. Either being part of the general refactor moving elements to `script setup` or when that already happened, an adjustment afterwards, like here.

### CODE REVIEW

File: packages/client/src/scripts/url.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export function query(obj: {}): string {
 		.reduce((a, [k, v]) => (a[k] = v, a), {} as Record<string, any>);
 
 	return Object.entries(params)
-		.map((e) => `${e[0]}=${encodeURIComponent(e[1])}`)
+		.map((p) => `${p[0]}=${encodeURIComponent(p[1])}`)
```
Comment (Johann150) [inline packages/client/src/scripts/url.ts:7]: I'm not sure if this is much of an improvement.

**Review by Johann150 — APPROVED** [review]
This also seems good to me.

File: packages/client/src/scripts/url.ts (hunk 1 above)
Comment (syuilo) [reply packages/client/src/scripts/url.ts:7]: `e` is confusing because it is often used as an abbreviation for `error`.

**Comment by syuilo** [comment]
🙏

**Pull request merged**